frontend/src/components/DeploymentDialog.vue

```diff
@@ -76,12 +76,13 @@ export default class DeploymentDialog extends Vue {
     }
 
     pushAutoScroll (...output) {
```

Comment:
No `...` required here. Output is an array.

Reply:
`output` is not an array when `pushAutoScroll` is called from `onFinish` (line 69), and the output gets destructed into characters when the triple dots on 78 (and 75) are removed: 
![image](https://user-images.githubusercontent.com/46179555/50565965-4b8ae400-0d02-11e9-81e9-ad78fe7efadc.png)

So if we want `pushAutoScroll` to assume it receives an array of strings, line 69 and any other calls to `pushAutoScroll` must surround single strings with brackets or else the above will occur. I don't think this is very straightforward and it will probably cause issues later on.
`this.pushAutoScroll([ 'Deployment finished.' ]);`

Alternatively, we could add the following to the top of `pushAutoScroll`, which does the conversion to an array automatically if it only receives a single object:
```
        if (Array.isArray(output) === false) {
            output = [ output ];
        }
```

Let me know your thoughts.

Reply:
Oh dear, this is a bug. Thanks for spotting this.

Reply:
Yes, I see it. Sorry, I was totally confused about this problem until I opened the code on my laptop. Initially I thought we're receiving a last line form backend, but this is not the case.

I'm generally against doing any magic with arguments. Let's make the function accept array only.

Reply:
OK, no problem. I'll make another commit with the other changes + this one.